### PR TITLE
Add the ability to customize each change line

### PR DESCRIPTION
**Pull request by toolmantim**
As requested in #35, this adds new `change-template` and `no-changes-template` configuration variables so you can completely customize the format for the release body.

- [x] Spec
- [x] Code
- [x] Tests

### CODE REVIEW

**Comment by toolmantim** [comment]
How are these new config options looking so far @salmanulfarzy?

**Comment by salmanulfarzy** [comment]
Honestly, Wasn't expecting a PR so quick and these options looks great :+1:

**Comment by toolmantim** [comment]
🙌🏼

**Comment by toolmantim** [comment]
This is live in [v2.1.0 (😴 Sleepy Time)](https://github.com/toolmantim/release-drafter/releases/tag/v2.1.0) and in the GitHub app.

File: README.md [hunk 1]
```diff
@@ -32,26 +32,39 @@ As pull requests are merged, a draft release is kept up-to-date listing the chan
...
-|`$PREVIOUS_TAG`|The previous releases’s tag.|
-
 ## Configuration options
 
 You can configure Release Drafter using the following key in your `.github/release-drafter.yml` file:
 
 |Key|Required|Description|
 |-|-|-|
 |`template`|Required|The template for the body of the draft release. Use [template variables](#template-variables) to insert values.|
+|`change-template`|Optional|The template to use for each merged pull request. Use [change template variables](#change-template-variables) to insert values. Default: `* $TITLE ($NUMBER) @$AUTHOR_USERNAME`|
```
Comment (salmanulfarzy) [inline README.md]: I think you missed to update `$AUTHOR_USERNAME` -> `$AUTHOR`

Comment (salmanulfarzy) [inline README.md]: I think you missed to update `$AUTHOR_USERNAME` -> `$AUTHOR` here.

**Comment by salmanulfarzy** [comment]
Awesome 😄

**Pull request merged**